modules/administration/pages/maintenance-window.adoc

```diff
@@ -0,0 +1,17 @@
+[[maintenance-window]]
+= Maintenance Window Checklist
+
+If you work on a system with scheduled maintenance windows, you might find it difficult to remember all the things that you need to during that critical downtime.
+
+This section contains a checklist for your maintenance window, with links to further information on performing each of the steps, if you need it.
+
+
+. Stop services.
+. Check if the configuration is still correct, and apply the highstate.
```

Comment:
applying highstate only makes sense if you maintain the suma server with Salt tool.